azure-pipelines.yml

```diff
@@ -104,9 +105,165 @@ stages:
           MIN_WORKER: $(INSTANCE_NUMBER)
           MAX_WORKER: $(INSTANCE_NUMBER)
           KVM_IMAGE_BRANCH: $(BUILD_BRANCH)
-          MGMT_BRANCH: $(BUILD_BRANCH)
+          MGMT_BRANCH: "master"
           STOP_ON_FAILURE: "False"
 
+  - job: impacted_area_t0_2vlans_elastictest
+    displayName: "impacted-area-kvmtest-t0_2vlans by Elastictest - optional"
+    dependsOn: get_impacted_area
+    condition: contains(dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS'], '2vlans')
```

Comment:
try eq(var1, var2) ?
![image](https://github.com/user-attachments/assets/6523de1f-d5c7-4a05-bfd1-581e189b234e)

Reply:
“eq” is used to determine whether two strings are equal. In our case, we need to determine whether one string contains another string or one string in another string. As the value `dependencies.get_impacted_area.outputs['SetVariableTask.PR_CHECKERS']` only can be a string, so we refrain from using overlapping string

Reply:
As we discussed, we will add a suffix for t0 and t1.